### ISSUE TITLE
Object detection error on tensorflow 1.0, Python3.5, Anaconda and WIndows10

### ISSUE THREAD

**Issue by pooyan1983** [tensorflow/models#2757]
When I run train.py in object detection package, I get the following errors. Not sure what Im missing. I have done all the configurations which are needed. Then when I run: python train.py --logtostderr --train_dir=training/ --pipeline_config_path=ssd_mobilenet_v1_pets.config 

I get the following error: 
Traceback (most recent call last):
  File "train.py", line 158, in <module>
    tf.app.run()
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\platform\app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "train.py", line 154, in main
    worker_job_name, is_chief, FLAGS.train_dir)
  File "C:\Anaconda\envs\tensorflow\models\research\object_detection\trainer.py", line 210, in train
    train_config.prefetch_queue_capacity, data_augmentation_options)
  File "C:\Anaconda\envs\tensorflow\models\research\object_detection\trainer.py", line 56, in create_input_queue
    tensor_dict = create_tensor_dict_fn()
  File "C:\Anaconda\envs\tensorflow\models\research\object_detection\builders\input_reader_builder.py", line 61, in build
    min_after_dequeue=input_reader_config.min_after_dequeue)
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\contrib\slim\python\slim\data\parallel_reader.py", line 214, in parallel_read
    name='filenames')
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\training\input.py", line 216, in string_input_producer
    string_tensor = ops.convert_to_tensor(string_tensor, dtype=dtypes.string)
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\framework\ops.py", line 639, in convert_to_tensor
    as_ref=False)
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\framework\ops.py", line 704, in internal_convert_to_tensor
    ret = conversion_func(value, dtype=dtype, name=name, as_ref=as_ref)
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\framework\constant_op.py", line 113, in _constant_tensor_conversion_function
    return constant(v, dtype=dtype, name=name)
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\framework\constant_op.py", line 102, in constant
    tensor_util.make_tensor_proto(value, dtype=dtype, shape=shape, verify_shape=verify_shape))
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\framework\tensor_util.py", line 370, in make_tensor_proto
    _AssertCompatible(values, dtype)
  File "C:\Anaconda\envs\tensorflow\lib\site-packages\tensorflow\python\framework\tensor_util.py", line 302, in _AssertCompatible
    (dtype.name, repr(mismatch), type(mismatch).__name__))
TypeError: Expected string, got ['data/train.record'] of type 'RepeatedScalarFieldContainer' instead.

**Comment by angersson** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by smitshilu** [comment]
Same error here. Did you find an answer?

**Comment by pooyan1983** [comment]
Yes I managed to solve this. The problem is with input_reader_builder module in object_detection\builders folder. In the build definition, there is a  line which reads
  _, string_tensor = parallel_reader.parallel_read(config.input_path,.......

For some reason, config.input_path does not create a string but gives back ['data/train.record'] so I just fixed that parameter as "data/train.record". This solved the problem for me.
